deployments/gitea.go

```diff
@@ -188,7 +188,17 @@ gitea:
     oauth2:
       ENABLE: true
       JWT_SECRET: HLNn92qqtznZSMkD_TzR_XFVdiZ5E87oaus6pyH7tiI
```

Comment:
yet another hardcoded secret? I also see and `INTERNAL_TOKEN` some lines above.

Reply:
oh, it's already in the description of #61 . So, work in progress.

Reply:
Yes. However it is unclear where they are used. I originally them linked to the regular gitea credentials, i.e. derived from them. Apparently not, as changing the gitea user/password does not require these to change to keep things working.

We could change/remove them and then see what breaks, if any.

Reply:
These are passed directly to gitea as config (https://docs.gitea.io/en-us/config-cheat-sheet/#oauth2-oauth2).

Reply:
Do we actually use the oauth parts of gitea ?

Reply:
See also https://docs.gitea.io/en-us/config-cheat-sheet/#security-security for the `INTERNAL_TOKEN`.

Reply:
Now testing which of these we can do without.